release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1125,6 +1125,84 @@ With this release, the CCO no longer checks for the nonexistent permission.
+
+* Previously, when editing a {vmw-full} connection, the form could be submitted even if no values were actually changed. This lead to the nodes rebooting. With this release, the console now detects the form changes, and does not allow submission if no value was changed. (link:https://issues.redhat.com/browse/OCPBUGS-31613[*OCPBUGS-31613*])
+
+* Previously, the `NetworkAttachmentDefinition` was always created in the default namespace if the form method `from the console` was used. The selected name is also not honored, and creates the `NetworkAttachmentDefinition` object with the selected name and a random suffix. With this release, the `NetworkAttachmentDefinition` object is created in the current project. (link:https://issues.redhat.com/browse/OCPBUGS-31558[*OCPBUGS-31558*])
+
+* Previously, when clicking the *Configure* button by the `AlertmanagerRecieversNotConfigured` alert, the *Configuration* page did not show. With this release, the link in the `AlertmanagerRecieversNotConfigured` alert is fixed and directs you to the *Configuration* page. (link:https://issues.redhat.com/browse/OCPBUGS-30805[*OCPBUGS-30805*])
+
+* Previously, plugins using `ListPageFilters` were only using two filters: label and name. With this release, a parameter was added that enables plugins to configure multiple text-based search filters. (link:https://issues.redhat.com/browse/OCPBUGS-30077[*OCPBUGS-30077*])
+
+* Previously, there was no response when clicking on quick start items. With this release, the quick start window shows when clicking on the quick start selections. (link:https://issues.redhat.com/browse/OCPBUGS-29992[*OCPBUGS-29992*])
+
+* Previously, the {product-title} web console terminated unexpectedly if authentication discovery failed on the first attempt. With this release, authentication initialization was updated to retry up to 5 minutes before failing. (link:https://issues.redhat.com/browse/OCPBUGS-29479[*OCPBUGS-29479*])
+
+* Previously there was an issue causing an error message on the *Image Manifest Vulnerability* page after an Image Manifest Vulnerability (IMV) was created in the CLI. With this release, the error message no longer shows. (link:https://issues.redhat.com/browse/OCPBUGS-28967[*OCPBUGS-28967*])
+
+* Previously, when using the modal dialog in a hook as part of the actions hook, an error occurs because the console framework will pass null objects as part of the render cycle. With this release, `getGroupVersionKindForResource` is now null safe and will return `undefined` if the `apiVersion` or `kind` are undefined. Additionally, the run time error for `useDeleteModal` no longer occurs, but note that it will not work with an `undefined` resource. (link:https://issues.redhat.com/browse/OCPBUGS-28856[*OCPBUGS-28856*])
```

Comment:
```suggestion
* Previously, when using the modal dialog in a hook as part of the actions hook, an error occurred because the console framework passed null objects as part of the render cycle. With this release, `getGroupVersionKindForResource` is now null-safe and will return `undefined` if the `apiVersion` or `kind` are undefined. Additionally, the run time error for `useDeleteModal` no longer occurs, but note that it will not work with an `undefined` resource. (link:https://issues.redhat.com/browse/OCPBUGS-28856[*OCPBUGS-28856*])
```